modules/asr/intent_engine/intent_engine.c

```diff
@@ -169,6 +170,8 @@ void intent_engine_task(void *args)
         if (asr_error != ASR_OK) continue;
 
         asr_error = asr_get_result(asr_ctx, &asr_result);
+        memcpy(&last_asr_result, &asr_result, sizeof(asr_result_t));
```

Comment:
`last_asr_result` is set here.  Where is it read?

Reply:
This struct is read in i2c_reg_handling.c.